### ISSUE TITLE
Command line arguments to choose the sender address and skip the dialog

### ISSUE THREAD

**Issue by TobiasKnauss** [speedball2001/identitychooser-mx#73]
Hello,
is there a way to automatically choose the sender email adress? I use Thunderbird's command line arguments to compose emails automatically (using VBA in MS Access), and I also provide the account ID of the email account from which I want to send the email:
commandLine = thunderbirdExePath _
            & " -compose format=2" _
            & ",preselectid=id" & thunderbirdEmailAccountID _
            & ",to=" & Replace(i_to, " ", "") _
            & commandLineCC _
            & commandLineBCC _
            & ",subject='" & i_subject & "'" _
            & ",body='" & i_body & "'" _
            & commandLineAttach

But then, Identity Chooser steps in and asks for the sender address, which makes the given account ID useless. And today I chose the the wrong sender address, so I would like to reestablish the automatism here again.

There are different possibilities:
- add a command line switch that skips Identity Chooser.
- add a command line switch that tells Identity Chooser which address to use
- Identity Chooser could evaluate the given command line arguments (accound ID) and select to correct account.

I am a software developer myself (C#, C++, VBA, etc), so I could give you feedback on a possible implementation.
Thanks in advance!

Übrigens: Es ist ziemlich schwierig, Sie zu kontaktieren. Auf der Webseite (https://addons.thunderbird.net/de/thunderbird/addon/identity-chooser/) zu Ihrem Addon schreiben Sie: "Sie erreichen mich über den Email-Link auf der rechten Seite.". Leider ist da kein E-Mail-Link.
Dann dachte ich, dass dieser Text vielleicht eine Kopie aus der Addon-Beschreibung ist, wie sie auch in den Thunderbird-Addon-Einstellungen steht und sich dort auf einen Link bezieht, aber da war auch keiner. Zum Glück stand da der Link auf das Github-Repo. ;-)

**Comment by speedball2001** [comment]
I don't think that this is possible. Mail Extensions such as Identity Chooser don't have any access to Thunderbird's command line arguments. 

If you want to manage the sender email address from the outside then you may have to deactivate Identity Chooser completely, I'm afraid.
